### PR TITLE
Add Jira API token as a secret.

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

**Review by martina-if — APPROVED** [review]
before we merge this, have you tested it somewhere? if not, I can help with that.

**Comment by Xantier** [comment]
> before we merge this, have you tested it somewhere? if not, I can help with that.

Yup, would be good to have a hand with testing this. I guess we can use roadie.roadie to do that.

**Review by martina-if — APPROVED** [review]

File: backstage/values.yaml [hunk 1]
```diff
@@ -149,6 +153,8 @@ appConfig:
     baseUrl: https://demo.example.com/lighthouse-api
   rollbar:
     organization: roadie
+  jiraProxy:
+    enabled: false
```
Comment (martina-if) [inline backstage/values.yaml]: btw, we might want to have this enabled by default

Comment (Xantier) [inline backstage/values.yaml]:
```suggestion
    enabled: true
```

Comment (Xantier) [reply backstage/values.yaml]: :+1: enabled by default

**Review by martina-if — APPROVED** [review]

**Pull request merged**